src/cloud.c

```diff
@@ -375,37 +377,27 @@ int cloud_auth_device(const char *id, const char *token)
  */
 int cloud_update_schema(const char *id, struct l_queue *schema_list)
 {
-	amqp_bytes_t queue_cloud;
 	json_object *jobj_schema;
 	const char *json_str;
 	int result;
 
-	queue_cloud = mq_declare_new_queue(MQ_QUEUE_FOG_IN);
-	if (!queue_cloud.bytes) {
-		l_error("Error on declare a new queue");
-		return -1;
-	}
-
 	jobj_schema = parser_schema_create_object(id, schema_list);
-	if (!jobj_schema) {
-		amqp_bytes_free(queue_cloud);
+	if (!jobj_schema)
 		return KNOT_ERR_CLOUD_FAILURE;
-	}
+
 	json_str = json_object_to_json_string(jobj_schema);
 
 	headers[0].value.value.bytes = amqp_cstring_bytes(user_auth_token);
 
-	result = mq_publish_persistent_message(queue_cloud,
-					       MQ_EXCHANGE_FOG_IN,
-					       MQ_CMD_SCHEMA_UPDATE,
-					       headers, 1,
-					       MQ_MSG_EXPIRATION_TIME_MS,
-					       json_str);
+	result = mq_publish_direct_message(MQ_EXCHANGE_DEVICE,
+					   MQ_CMD_SCHEMA_SENT,
+					   headers, 1,
```

Comment:
Add a comment informing whats is 1 or why it should be 1.

Reply:
I put a disclaimer informing about the AMQP settings of the sent message